morphuntion/resources/org/unicode/morphuntion/inflection/pronoun.properties

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright 2022-2024 Apple Inc. All rights reserved.
+#
+yue_HK=yue_Hant
```

Comment:
Is this just mapping one form of LangID to another? Why (comment)?

Reply:
It's an aliasing scheme. In this case, we're making the script explicit for a given region.